README.md

```diff
@@ -2,20 +2,30 @@
 
 Printing is a common and important task in many domains, including document management, label printing, receipt printing, and more. While web applications can generate printable content such as PDFs, images, and text documents, sending print jobs from web applications to local printers has traditionally required various workarounds such as relying on third-party plugins or polyfilling via `window.print()`; `navigator.printing` API aims to provide a standardized and seamless way for web applications to directly interact with printers, enabling developers to build printer-related features within their web applications.
 
 The explainer outlines the motivating use cases for the API, its key features, and how it can benefit web developers and users alike.
 
 ## Motivating Use Cases
 
-### Augmenting `window.print()`
+Although `window.print()` offers basic printing functionality through the browser's default print dialog, it is a relatively simple API. Developers' capabilities are restricted to selecting a file for printing and triggering the print dialog, which places the decision-making responsibility solely on the end user. To address these limitations and facilitate advanced use cases, we aim to provide developers with greater control over the printing process and expand the range of options available to them.
 
-While `window.print()` is a basic function that triggers the browser's default print dialog, the `navigator.printing` API provides a more comprehensive set of methods and options for querying local printers, submitting print jobs with customization options, managing print jobs, and customizing print workflows. This extended functionality allows developers to have more control over the printing process and enables advanced use cases, such as printing specific pages or sections of a document, setting print options like paper size and orientation, and monitoring the status of print jobs.
+### Customizing Print Settings
+The enhanced features aim to give developers greater influence over the printing process of a document. These capabilities include selecting the printer without requiring user input, printing specific pages or sections of the document, and adjusting print settings such as paper size, orientation, or finishings directly through the script.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Knowing Your Printers
```

Comment:
To the point above, I would provide examples of the user benefits of giving applications information about the printer. How does it enhance the experience?

Reply:
Added a few examples.